src/ui/starter-select-ui-handler.ts

```diff
@@ -1725,6 +1725,7 @@ export default class StarterSelectUiHandler extends MessageUiHandler {
                     });
                     ui.setMode(Mode.STARTER_SELECT);
                     this.setSpeciesDetails(this.lastSpecies, undefined, undefined, undefined, undefined, undefined, undefined);
```

Comment:
```suggestion
                    this.setSpeciesDetails(this.lastSpecies);
```
"Better fill in all these optional parameters" 🤔 
Might as well fix this while you're adding a line directly below it lol.